mimic/rest/loadbalancer_api.py

```diff
@@ -26,20 +27,20 @@
 @implementer(IAPIMock, IPlugin)
 class LoadBalancerApi(object):
     """
-    Rest endpoints for mocked Load balancer api.
+    This class registers the load balancer API in the service catalog.
+    It does NOT implement individual endpoints.
```

Comment:
I'm curious why you felt it necessary to call out that this doesn't implement individual endpoints here; it seems much like every other plugin in that regard.  Should we add something to more central documentation about that, so we can refer to it from many plugins?